techmatters/terraso-web-client
# fix: use distinct text when leaving landscapes and groups

## Description
Pass in custom text and title to the confirmation dialog for leaving groups and landscapes.

### Checklist

### Related Issues
Fixes #160.

### Verification steps
1. Go to a group, i.e. http://127.0.0.1:3000/groups/acus-penninsula-galera-san-francisco
2. Click leave
3. Confirm group-specific language
4. Go to a landscape, i.e. http://127.0.0.1:3000/landscapes/acus-penninsula-galera-san-francisco
5. Click leave
6. Confirm landscape-specific language
